packages/rhsm/setup.cfg

```diff
@@ -0,0 +1,4 @@
+[nosetests]
+where=../../test/rhsmlib_test
```

Comment:
Directory: `../../test/rhsmlib_test` contains unit tests for `rhsmlib` module. Unit tests for `rhsm` module are in `../../test/rhsm/unit`.

Reply:
Anyway, I wasn't able to run test for rhsm.